libcontainer/setns_init_linux.go

```diff
@@ -135,7 +135,7 @@ func (l *linuxSetnsInit) Init() error {
 		return &os.PathError{Op: "close log pipe", Path: "fd " + strconv.Itoa(l.logFd), Err: err}
 	}
 
-	if l.dmzExe != nil {
```

Comment:
This approach re-introduces the CVEs that `runc-dmz` is protecting against.

In order to take this approach, you will need to modify `isDmzBinarySafe` to detect this problem before we do `runc init`. Effectively you need to make a check that mirrors the `dmz.WorksWithSELinux` check.